docs/advanced-topics/docker-alternatives.rst

```diff
@@ -62,28 +62,30 @@ with Dockstore by adding the following line to your ``~/.dockstore/config``:
 
 This may not work with all CWL entries, but it is a good workaround for the cwltool incompatibility described above.
 
-Singularity
------------
+Singularity (also known as SingularityCE or Apptainer)
+-------------------------------------------------------
 
-Singularity is perhaps the most well-supported Docker alternative. Singularity can pull Docker images and build them
-into its own image format (.sif), but not all Docker features are compatible. For instance, dockerfile ``USER``
-commands are not compatible with Singularity.
+Singularity in its various incarnations as Apptainer or SingularityCE are perhaps the most well-supported Docker alternative(s).
```

Comment:
Nit: if we bracket the optional descriptive phrase with commas, this might be easier to read.  After making this change, it seems like, arguably, the verb tense should remain singular.  That is: 
```
Singularity, in its various incarnations as Apptainer or SingularityCE, is perhaps the most well-supported the most well-supported Docker alternative.
```